modules/cli-installing-cli.adoc

```diff
@@ -98,7 +98,7 @@ You can install the OpenShift CLI (`oc`) binary on Windows by using the followin
 .Procedure
 
 ifdef::openshift-origin[]
-. Navigate to https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/ and choose the folder for your operating system and architecture.
+. Navigate to https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/ and choose the folder for your operating system and architecture.
```

Comment:
This procedure is actually for installing on Windows, so this x86_64 link (which is for Linux), isn't appropriate.

And looking at the deprecation notice in that original location, only the x86_64 version for the Linux procedure is deprecated. So I think you can revert this update completely.